easybuild/easyconfigs/f/fastPHASE/fastPHASE-1.4.8.eb

```diff
@@ -0,0 +1,27 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Ravi Tripathi
+# Email: ravi89@uab.edu
+
+easyblock = "BinariesTarball"
+
+name = 'fastPHASE'
+version = '1.4.8'
+
+homepage = 'http://stephenslab.uchicago.edu/software.html#fastphase'
+description = """ fastPHASE: software for haplotype reconstruction, 
+and estimating missing genotypes from population data
+Documentation: http://scheet.org/code/fastphase_doc_1.4.pdf"""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+source_urls = ['http://scheet.org/code/']
+sources = ['Linuxfp.tar.gz']
+
+modextrapaths = {'PATH': ''}
```

Comment:
@rtripath89 this looks wrong, why is this needed if the binary is in a `bin` subdirectory (which is added to `$PATH` automagically by EB)